src/simplewallet/simplewallet.cpp

```diff
@@ -1838,25 +1911,28 @@ bool simple_wallet::show_incoming_transfers(const std::vector<std::string>& args
   {
     if (!filter || available != td.m_spent)
     {
+      if (m_wallet->get_current_subaddress_account() != td.m_subaddr_index.major || (!subaddr_indices.empty() && subaddr_indices.count(td.m_subaddr_index.minor) == 0))
+        continue;
```

Comment:
Correct, that's my design intention: incoming/outgoing transfers are completely separated per account, and the user needs to switch accounts to inspect them. Do you think it's necessary to allow the user to print the whole set?